src/connectionController.ts

```diff
@@ -603,7 +603,7 @@ export default class ConnectionController {
       'mdb.isAtlasStreams',
       false
     );
-    void vscode.window.showInformationMessage('MongoDB disconnected.');
+    this._statusView.showMessage('MongoDB disconnected.');
 
     this._disconnecting = false;
     this._statusView.hideMessage();
```

Comment:
Nice catch! I think we can just remove the hiding since there is not so much you can do when disconnected. And we replace it with the connected status when there is an active connection.